ui/admin/app/routes/onboarding.js

```diff
@@ -25,6 +26,8 @@ export default class OnboardingRoute extends Route {
   async beforeModel() {
     const orgs = await this.store.query('scope', {
       scope_id: 'global',
+      recursive: true,
```

Comment:
One last small thing 🙏 , can we actually make the requests match what they were previously, i.e. if they were using global scope and recursive, send both but if they were just using global scope but not recursive, don't include the recursive query param? 

As part of the changes I'm making to how we query, the params matter and the query filter below doesn't recursively return all the data anyways. We don't have an easy way to return data recursively without doing the filters manually right now

Reply:
Of course! Updated!